app/views/support_interface/application_forms/immigration_status/edit.html.erb

```diff
@@ -8,29 +8,42 @@
       <% if @immigration_status_form.eu_nationality? %>
         <%= f.govuk_radio_buttons_fieldset(
         :immigration_status,
-        legend: { text: 'Edit applicant immigration status', size: 'l' },
+        legend: { text: t('support_interface.page_titles.immigration_status'), size: 'l' },
       ) do %>
-        <%= f.govuk_radio_button :immigration_status, 'eu_settled', label: { text: 'EU settled status' }, link_errors: true %>
-        <%= f.govuk_radio_button :immigration_status, 'eu_pre_settled', label: { text: 'EU pre-settled status' } %>
-        <%= f.govuk_radio_button :immigration_status, 'other', label: { text: 'Other' } do %>
-          <%= f.govuk_text_area(
+        <%= f.govuk_radio_button :immigration_status, :eu_settled, label: { text: t('support_interface.edit_immigration_status.values.eu_settled') }, link_errors: true %>
+        <%= f.govuk_radio_button :immigration_status, :eu_pre_settled, label: { text: t('support_interface.edit_immigration_status.values.eu_pre_settled') } %>
+        <%= f.govuk_radio_divider 'or' %>
+        <%= f.govuk_radio_button :immigration_status, :other, label: { text: t('support_interface.edit_immigration_status.values.other') } do %>
+          <%= f.govuk_text_field(
                 :right_to_work_or_study_details,
-                label: { text: 'Immigration status', size: 's' },
-                hint: { text: 'For example, “Indefinite leave to remain”' },
-                max_words: 200,
-                threshold: 90,
+                label: { text: 'Enter Immigration status' },
               ) %>
         <% end %>
       <% end %>
       <% else %>
-        <%= f.hidden_field :immigration_status, value: 'other' %>
-        <%= f.govuk_text_area(
-              :right_to_work_or_study_details,
-              label: { text: 'Immigration status', size: 'l' },
-              hint: { text: 'For example, “I have permanent residence”' },
-              max_words: 200,
-              threshold: 90,
-            ) %>
+      <h1 class="govuk-heading-l"><%= t('support_interface.page_titles.visa_or_immigration_status') %></h1>
+        <%= f.govuk_radio_buttons_fieldset :right_to_work_or_study_details,
+        legend: { text: 'Select your visa or immigration status' } do %>
+          <%= f.govuk_radio_button :immigration_status, :indefinite_leave_to_remain_in_the_uk, label: { text: t('support_interface.edit_immigration_status.values.indefinite_leave_to_remain_in_the_uk') } %>
```

Comment:
Same comment about the `scope` approach to reduce the repetition and make the markup easier to scan.

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/9012/commits/c23d80bfbb7924618fd4ddb95634838f14b8d1c4